projects/objects/balls/protos/RobocupSoccerBall.proto

```diff
@@ -5,18 +5,24 @@
 # This object is physics-enabled so it will roll or bounce when hit.
```

Comment:
It would be nice to add in the description the specification of the radius and mass for each ball size given that the size doesn't describe the radius.